app/src/utils/FactionUtils.ts

```diff
@@ -0,0 +1,20 @@
+import {Faction} from '../constants/faction';
+import IllegalArgumentException from '../exceptions/IllegalArgumentException';
+
+export default class FactionUtils {
+    public static parse(value: number): Faction {
+        switch (value) {
+            case Faction.none:
+                return  Faction.none;
+            case Faction.newConglomerate:
+                return Faction.newConglomerate;
+            case Faction.nSOperatives:
+                return Faction.nSOperatives;
```

Comment:
Being a real picky bastard here but could we be a little more verbose and call it `naniteSystemsOperatives`?

Reply:
http://census.daybreakgames.com/s:CENSUS/get/ps2/faction?c:limit=500 reports them as NS Operatives.
I used the english version

Reply:
I get it, we don't need to strictly map it 1:1, just reads better imo especially considering the rest are also "verbosely" named, e.g. not `vSoverignty`.

Bad naming by PS2 Devs!

Reply:
I feel it may help other people understand it better, if we keep it close to the census api (maybe with a comment mentioning the API as source?)

Reply:
Alright, based on that logic I'll concede, I perfer to name my vars as reasonably verbose as realistically possible but considering in other places we've matched the API I'll drop it :+1: